src/index.njk

```diff
@@ -64,8 +64,9 @@ description: Design your service using GOV.UK styles, components and patterns
           </a>
         </li>
         <li>
-          <a href="https://ukgovernmentdigital.slack.com/messages/govuk-design-system" class="govuk-link" data-hsupport="slack">
-            get in touch on Slack</a> (<a class="govuk-link" href="slack://channel?team=T04V6EBTR&amp;id=C6DMEH5R6" data-hsupport="slackapp">open in app</a>)
```

Comment:
@Daisy25 just for your info, Ollie is removing a link that we are tracking currently.